tests/CustomerBundle/DependencyInjection/SonataCustomerExtensionTest.php

```diff
@@ -0,0 +1,144 @@
+            'sonata.customer.profile.menu_builder.default'
+        );
+    }
+
+    public function testTwigConfigParameterIsSetting(): void
+    {
+        $fakeContainer = $this->getMockBuilder(ContainerBuilder::class)
+            ->setMethods(['hasExtension', 'prependExtensionConfig'])
+            ->getMock();
+
+        $fakeContainer->expects($this->once())
+            ->method('hasExtension')
+            ->with($this->equalTo('twig'))
+            ->willReturn(true);
+
+        $fakeContainer->expects($this->once())
+            ->method('prependExtensionConfig')
+            ->with('twig', ['form_themes' => ['@SonataCore/Form/datepicker.html.twig']]);
+
+        foreach ($this->getContainerExtensions() as $extension) {
+            if ($extension instanceof PrependExtensionInterface) {
+                $extension->prepend($fakeContainer);
+            }
+        }
+    }
+
+    /**
+     * @group legacy
+     */
+    public function testTwigConfigParameterIsSet(): void
+    {
+        $fakeTwigExtension = $this->getMockBuilder(TwigExtension::class)
+            ->setMethods(['load', 'getAlias'])
+            ->getMock();
+
+        $fakeTwigExtension
+            ->method('getAlias')
+            ->willReturn('twig');
+
+        $this->container->registerExtension($fakeTwigExtension);
+
+        $this->load();
+
+        $twigConfigurations = $this->container->getExtensionConfig('twig');
+
+        $this->assertArrayHasKey(0, $twigConfigurations);
+        $this->assertArrayHasKey('form_themes', $twigConfigurations[0]);
+        $this->assertSame(
+            ['@SonataCore/Form/datepicker.html.twig'],
+            $twigConfigurations[0]['form_themes']
+        );
+    }
+
+    /**
+     * @group legacy
+     */
+    public function testTwigConfigParameterIsNotSet(): void
+    {
+        $this->load();
+
+        $twigConfigurations = $this->container->getExtensionConfig('twig');
+
+        $this->assertArrayNotHasKey(0, $twigConfigurations);
+    }
+
+    /**
+     * @group legacy
+     */
+    public function testEmptyProfile(): void
+    {
+        $this->load([
+            'profile' => [
+                'menu' => null,
+                'blocks' => null,
+            ],
+        ]);
+    }
+
+    protected function getMinimalConfiguration(): array
+    {
+        return (new Processor())->process((new Configuration())->getConfigTreeBuilder()->buildTree(), []);
+    }
+
+    protected function getContainerExtensions(): array
+    {
+        return [
+            new SonataCustomerExtension(),
+        ];
+    }
+
+    protected function load(array $configurationValues = []): void
+    {
+        $configs = [$this->getMinimalConfiguration(), $configurationValues];
+
+        foreach ($this->container->getExtensions() as $extension) {
+            if ($extension instanceof PrependExtensionInterface) {
+                $extension->prepend($this->container);
+            }
+
+            $extension->load($configs, $this->container);
```

Comment:
Is required to call `load()` on every iteration?

Reply:
Yes, some test have diffrence configs. Check methods: `testTwigConfigParameterIsNotSet` and `testEmptyProfile`.